src/views/templates/list/hooks/useWorkloadProfile.ts

```diff
@@ -6,18 +6,19 @@ import { getTemplateWorkload } from '../../utils/selectors';
 // t('plugin__kubevirt-plugin~Desktop')
 // t('plugin__kubevirt-plugin~Server')
 // t('plugin__kubevirt-plugin~High performance')
-// t('plugin__kubevirt-plugin~Other')
 
-const workloadLabels = {
+export const workloadLabels = {
   desktop: 'Desktop',
   server: 'Server',
   highperformance: 'High performance',
 };
 
-const useWorkloadProfile = (template: V1Template): string => {
+export type WorkloadProfileNames = 'Desktop' | 'Server' | 'High performance';
```

Comment:
```suggestion
export type WorkloadProfileNames = 'Desktop' | 'Server' | 'High performance';
export type WorkloadProfileKeys = 'desktop' | 'server' | 'highperformance';

```

Reply:
Choosing `WorkloadProfileNames` as the name of the type. No need to use `WorkloadProfileDisplayNames` because it is unnecessary long name, and it can be messed with the template's "Display name" which is something different.

Reply:
And `WorkloadProfileKeys` type [already added](https://github.com/kubevirt-ui/kubevirt-plugin/pull/477/files#diff-7e0e1100c332e878944a89ad9bb14ab21a3a7622063d562320fe14e05cab4423R18).